dvc/machine/__init__.py

```diff
@@ -200,3 +202,7 @@ def get_sshfs(self, name: Optional[str]):
     def run_shell(self, name: Optional[str]):
         config, backend = self.get_config_and_backend(name)
         return backend.run_shell(**config)
+
+    def status(self, name: Optional[str]) -> Iterator[Dict[Any, Any]]:
```

Comment:
perhaps move `SENSITIVE_FIELDS` to this function?

```suggestion
    def status(self, name: str, hide_sensitive: bool=True) -> Iterator[Dict[Any, Any]]:
```

Reply:
@karajan1001 this was just a stub - the `SENSITIVE_FIELDS` logic would have to be moved to this class

Reply:
Discussed this with peter yesterday. Here we use will use a white list to limit the showing fields to only in (name, cloud, instance size, IP address) for now. The upcoming `--show-json` can output the whole config in json format in the future.